pkg/vmprovider/providers/vsphere/constants/constants.go

```diff
@@ -114,4 +114,6 @@ const (
 	V1alpha1SubnetMask = "V1alpha1_SubnetMask"
 	// V1alpha1FormatNameservers is an alias for versioned templating function V1alpha1_FormatNameservers.
 	V1alpha1FormatNameservers = "V1alpha1_FormatNameservers"
+
+	ContentLibraryServiceTypeLabelKey = "contentlibrary.imageregistry.vmware.com/servicetype"
```

Comment:
@zyiyi11 If this is only used in the ccl-item controller test code above, `controllers/contentlibrary/utils/test_utils.go` might be a better place to store it.